### PR TITLE
Added an example for ClusterableModel in snippets

### PR DESCRIPTION
**Pull request by onno-timmerman**
My first document change. I hope This is doen correctly!

I found it missing a concrete example of Clusterable models with an Orderable

ClusterableModel snippets
-------------------------

If you want to group a snippet you can use ClusterableModel. For example you want to insert bands and there members.
The band member can then be orderd accordingly.

.. code-block:: python

    class BandMember(Orderable):
         band = ParentalKey('Band', related_name='members', on_delete=models.CASCADE)
         name = models.CharField(max_length=255)

    @register_snippet
    class Band(ClusterableModel):
         name = models.CharField(max_length=255)

         panels = [
             FieldPanel('name'),
             InlinePanel('members')
      ]

### CODE REVIEW

**Comment by zerolab** [comment]
Hey @onno-timmerman, you probably want to update the PR to have the target branch as `wagtail:master`, not `wagtail:3-month-release-doc-update`

File: .circleci/config.yml [hunk 1]
```diff
@@ -47,9 +47,35 @@ jobs:
       - run: npm run dist
       - run: bash <(curl -s https://codecov.io/bash) -F frontend
 
+  nightly-build:
```
Comment (lb-) [inline .circleci/config.yml]: It might be good to remove this file's changes from the PR as this is unrelated to the actual documentation changes.

File: .circleci/trigger-nightly-build.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```
Comment (lb-) [inline .circleci/trigger-nightly-build.sh]: same here, best to remove unrelated changes from this PR

File: .squash.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 deployments:
   default:
-    dockerimage: python:latest
+    dockerimage: python:3.7.4-stretch
```
Comment (lb-) [inline .squash.yml]: same again - updating this is probably not needed
